s2spy/time.py

```diff
@@ -224,20 +241,14 @@ def show(self) -> pd.DataFrame:
             pd.Dataframe: Dataframe containing the calendar intervals, with the target
                 periods labelled.
         """
-        return self._label_targets(self.get_intervals()).applymap(
-            self._interval_as_month
-        )
+        return self.get_intervals().applymap(self._interval_as_month)
 
 
-class WeeklyCalendar(BaseCalendar):
+class WeeklyCalendar(AdventCalendar):
```

Comment:
The docstring for "anchor" has not been updated in the WeeklyCalendar.

Reply:
The example in the docstring also seems to not be tested (anymore)?!

Try the example:
```
import s2spy.time
calendar = s2spy.time.WeeklyCalendar(anchor=40, freq="1W")
```

Reply:
I think the docstring check was not switched on before. I will turn it on and fix the docstrings. 😉